extension/src/reactor/index.ts

```diff
@@ -149,16 +149,20 @@ export class Reactor<
     };
   }
 
-  private async handleMainBreakpointEvent(
+  private async handleInitialBreakpointEvent(
     frameId: FrameId,
+    stopLocation: Location,
   ) {
-    // The program has stopped at main
+    // GDB: The program has stopped at main
+    // Debugpy: The program has stopped at the first breakpoint
     // Perform all initialization actions
     if (
       this.session instanceof GDBDebuggerSession &&
```

Comment:
Same here, you could just call a method on the session to handle the initial breakpoint event?